### ISSUE TITLE
feat: add collapse option to individual folder

### ISSUE THREAD

**Issue by mdobydullah** [ZTL-UwU/shadcn-docs-nuxt#50]
### Describe the feature

It would be nice to have the collapse option for individual folders.

## Example

app.config.ts
```ts
aside: {
  collapse: true,
}
```

_dir.yml
```yml
title: Configuration
collapse: false
```

![collapse](https://github.com/user-attachments/assets/78b95e79-40a2-41bd-a4a3-e5b61c6b5628)

### Additional information

- [ ] Would you be willing to help implement this feature?

**Comment by ZTL-UwU** [comment]
Added in 1b4fd75ec54904c98974e5f2383adb22fbfdb2ba.

Usage: https://shadcn-docs-nuxt.vercel.app/getting-started/writing/folders#parameters

**Comment by mdobydullah** [comment]
@ZTL-UwU Thank you so much. 🔥

**Comment by ZTL-UwU** [comment]
It seems that it does not override the global config. Will fix this later.

**Comment by mdobydullah** [comment]
Ah okay.

**Comment by ZTL-UwU** [comment]
Fixed in b2960ba.

p.s. I changed the field name to `collapse` so that it is consistent with `app.config`.

**Comment by mdobydullah** [comment]
Perfect. Thank you so much for quickly adding the option. ❤️
